src/main/java/de/dennisguse/opentracks/io/file/importer/KmlFileTrackImporter.java

```diff
@@ -186,9 +186,9 @@ private void onWaypointEnd() throws SAXException {
             return;
         }
 
+        // If there is photoUrl it has to be changed because that url in kml file isn't a valid content:// to the imported one.
         if (photoUrl != null) {
-            Uri uri = Uri.parse(photoUrl);
-            photoUrl = getPhotoUrl(uri.getLastPathSegment());
+            photoUrl = getPhotoUrl(Uri.parse(photoUrl).getLastPathSegment());
```

Comment:
Uri parsing might fail and there might be no path.

And as we are importing here something from somewhere, we should check this carefully.